src/components/MoviesList/MoviesList.js

```diff
@@ -1,6 +1,25 @@
 import React from 'react';
 import './MoviesList.scss';
+import PropTypes from 'prop-types';
+import { MovieCard } from '../MovieCard/MovieCard';
+import { TypeMovie } from '../../types';
 
-export const MoviesList = () => (
-  <>Put the list here</>
+export const MoviesList = ({ movies }) => (
+  <div className="movies">
+    {movies.map(movie => (
+      <div key={movie.imdbId} className="card">
+        <MovieCard {...movie} />
+      </div>
```

Comment:
It's the list of movies, so you'd better use tags UL and Li here instead of divs